app/js/lib/speech-controller.js

```diff
@@ -41,36 +43,39 @@ export default class SpeechController extends EventDispatcher {
   constructor() {
     super(EVENT_INTERFACE);
 
-    const recognizer = new WakeWordRecognizer();
+    const wakeWordRecogniser = new WakeWordRecogniser();
+    const speechRecogniser = new SpeechRecogniser();
 
-    recognizer.setOnKeywordSpottedCallback(
+    wakeWordRecogniser.setOnKeywordSpottedCallback(
       this[p.handleKeywordSpotted].bind(this));
 
-    this[p.wakewordRecognizer] = recognizer;
+    this[p.wakewordRecogniser] = wakeWordRecogniser;
     this[p.wakewordModelUrl] = '/data/wakeword_model.json';
```

Comment:
I just realised that this path may not work if the app is not at the domain root (e.g. `https://fxbox.github.io/calendar`).